flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/HybridShuffleConfiguration.java

```diff
@@ -135,8 +146,23 @@ public float getFullStrategyReleaseBufferRatio() {
         return fullStrategyReleaseBufferRatio;
     }
 
+    /** Get {@link HsSelectiveSpillingStrategy} for hybrid shuffle mode. */
+    public HsSpillingStrategy getSpillingStrategy() {
+        switch (spillingStrategyName.toLowerCase()) {
+            case FULL_SPILLING_STRATEGY_NAME:
+                return new HsFullSpillingStrategy(this);
+
+            case SELECTIVE_SPILLING_STRATEGY_NAME:
+                return new HsSelectiveSpillingStrategy(this);
+
+            default:
+                throw new IllegalConfigurationException(
+                        "Unknown spilling strategy: " + spillingStrategyName);
+        }
+    }
```

Comment:
As offline discuss, we will decide spilling strategy by `ResultPartitionType`(HYBRID_FULL & HYBRID_SELECTIVE). I think using string value to load spilling strategy just for temporary may also be acceptable, as it will be removed in next PR.

Reply:
done.